World.dm

```diff
@@ -562,23 +560,6 @@ client
 				mob.LastLoc = mob.loc
 				..()
 /proc
-	RuinAll()
```

Comment:
Manual deletion is more effective and wont throw any errors without. also modernizing